### ISSUE TITLE
WCAG 2 translations missing in W3C listing

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-website#923]
### Resource URL

https://www.w3.org/Translations/

### Description

Hi Rémi. I didn't look in detail, yet from quick skim, it looks like:
W3C list: https://www.w3.org/Translations/
is missing several that are listed at:
WAI list: https://www.w3.org/WAI/standards-guidelines/wcag/translations/

Would you work with @xueyuanjia to get the W3C list updated.

**Comment by remibetin** [comment]
@shawna-slh I have not spotted missing _completed_ translations for WCAG 2.1 and WCAG 2.2.

I see a few missing WCAG 2.0 translations. @xueyuanjia What information do you need to update W3C list?

**Comment by xueyuanjia** [comment]
@shawna-slh @remibetin The W3C list [https://www.w3.org/Translations/](https://www.w3.org/Translations/) now contains all completed translations from the WAI list [https://www.w3.org/WAI/standards-guidelines/wcag/translations/](https://www.w3.org/WAI/standards-guidelines/wcag/translations/), and the matrix page [https://www.w3.org/Consortium/Translation/matrix.html](https://www.w3.org/Consortium/Translation/matrix.html) lists (almost) all planned and completed translations. 

Note: there's a bug in the W3C translation database so the languages Portuguese (Portugal) and Slovak on the matrix page are not displayed correctly, I will report it to our systeam.

Regarding WCAG 2.0 translations, you may want to update the following titles to make them consistent with their documents:

* Dutch: from "Richtlijnen voor de Toegankelijkheid van Webcontent (WCAG) 2.0" to "Richtlijnen voor Toegankelijkheid van Webcontent (WCAG) 2.0"
*  Estonian: from "Veebi sisu juurdepääsetavussuunised (WCAG 2.0)" to "Veebi sisu juurdepääsetavussuunised (WCAG) 2.0"
* Russian: from "Руководство по обеспечению доступности Web-контента (WCAG) 2.0" to "Руководство по обеспечению доступности веб-контента (WCAG) 2.0"

Regarding WCAG 2.1 translations, would you like to include the following translations on the WAI list? 
* German (Unofficial translation) - 30 June 2022
** URI <https://outline-rocks.github.io/wcag/translations/WCAG21-de/>
** Translator: Aktion Mensch e.V.
** see <https://lists.w3.org/Archives/Public/w3c-translators/2022AprJun/0027.html>
* Russian
** Planned Unofficial Translation
** see: <https://lists.w3.org/Archives/Public/w3c-translators/2020OctDec/0024.html>

* Swedish
** Planned Authorized Translation
** Notification of intent (1 October 2018) <https://lists.w3.org/Archives/Public/w3c-translators/2018OctDec/0000.html>

* Turkish
** Notification of intent (11 July 2019) <https://lists.w3.org/Archives/Public/w3c-translators/2019JulSep/0011.html>

**Comment by remibetin** [comment]
@xueyuanjia Thanks for spotting these differences. I have opened a pull request: https://github.com/w3c/wai-website-data/pull/177

**Comment by xueyuanjia** [comment]
To follow up:
> Note: there's a bug in the W3C translation database so the languages Portuguese (Portugal), Slovak, Hindi, and Romanian do not appear on the matrix page correctly, I have reported it to our systeam.
> 

The missing languages tags now appear on the matrix page. \o/
